job_bundles/README.md

```diff
@@ -0,0 +1,315 @@
+[defined in the Job Template](https://github.com/aws-deadline/deadline-cloud-samples/blob/bdd5ff5ea29eb7457c9a78ba39166b891b79151e/job_bundles/blender_render/template.yaml#L11-L19) within 
+your Job Bundle, and the [Queue Environments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html)
+defined on the Queue that you are submitting to.
+
+If you prefer an alternative to a UI-based workflow for your Job Bundles, then you can submit this Job Bundle with the command
+`deadline bundle submit --name Demo -p BlenderSceneFile=<location-of-your-scene-file> -p OutputDir=<file-path-for-job-outputs> blender_render/`
+or use the `deadline.client.api.create_job_from_job_bundle` function in the [`deadline` Python package](https://github.com/aws-deadline/deadline-cloud).
+You can also develop your Job as a Job Template and use the
+[deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html) directly if you do not need
+to make use of a Job Bundle's integration with the Job Attachments feature.
+
+All of the Deadline Cloud submitters that have been developed by the AWS Deadline Cloud team, such as the 
+[Autodesk Maya submitter](https://github.com/aws-deadline/deadline-cloud-for-maya), are simply generating a Job Bundle for your
+Job submission and then using the [Deadline Cloud Python package](https://github.com/aws-deadline/deadline-cloud) to
+submit your Job to Deadline Cloud. You can inspect the job bundles created for previously submitted jobs by looking in the job history directory.
+You can find your job history directory by running the command: `deadline config get settings.job_history_dir`.
+
+## Elements of a Job Bundle
+
+A Job Bundle is a directory structure that contains at least an
+[Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) file, but may contain
+other files such as:
+
+```
+<BUNDLE_DIR>/
+├── asset_references.yaml (or asset_references.json)
+├── parameter_values.yaml (or parameter_values.json)
+├── template.yaml (or template.json)
+└── <other Job-specific files that you'd like>
+```
+
+The only required file is the Job Template (`template.yaml`/`template.json`) file that describes the structure and behaviour
+of your Job. The files are described in the following subsections.
+
+### Elements - Job Template
+
+The Job Template file defines the runtime environment and the processes that will run
+as part of an AWS Deadline Cloud Job. It can be parameterized so that the same template can be used to
+create Jobs that differ only in their input values; much like a function or template in your favourite programming
+langauge. 
+
+
+> **NOTE:** You can also setup [Open Job Description external environments](https://github.com/OpenJobDescription/openjd-specifications/wiki/2023-09-Template-Schemas#12-environment-template)
```

Comment:
This note feels out of place here. Maybe move this to the intro paragraph as a sentence like "When you submit a job to Deadline Cloud, it gets run within any queue environments that are applied to the queue. Queue environments use the [Open Job Description external environment specification](...).